yb-voyager/src/dbzm/config.go

```diff
@@ -304,7 +305,7 @@ func (c *Config) String() string {
 			triggerDirPath)
 		if c.SSLRootCert != "" {
```

Comment:
1. Don't think it'd be right to add sslmode only if the RootCert is passed. Find a way to pass in all the required config. (check the pg config code)
2. Let's throw an unsupported error if the user passes the key/cert in `export-data-from-target`

Reply:
> Let's throw an unsupported error if the user passes the key/cert in export-data-from-target
User can't pass these TargetConf related flags in the `export data from target`, It is same what is passed to import data case.

Reply:
Ah yes! Good point!
In that case, we're essentially ignoring those key/cert params. We should take explicit confirmation from the user that it's okay to ignore these ssl-params OR at the very least warn the user that we are ignoring. 

Since these are security-related params, we need to be careful.

Reply:
yeah, makes sense.

Reply:
1. Added a warning for the user if ssl cert and key are passed saying that they will be ignored.
2. The sslmode will be set only when rootcert is passed. Otherwise supplying an empty root cert raises an error.
3. Also made the `--target-ssl-mode` flag available in export data from target since the mode '`prefer`' is not supported by `yb debezium`. So, if the user ends up giving that mode in import data, after cutover an error will be raised to the user saying that '`prefer`' mode is not supported for yb and they can start the command with a new `--target-ssl-mode` flag.